internal/store/libkv.go

```diff
@@ -71,6 +71,60 @@ func (s *libKVStore) List(ctx context.Context, directory string) ([]*KVPair, err
 	return kvPairs, nil
 }
 
+func (s *libKVStore) Watch(ctx context.Context, key string) (<-chan *KVPair, error) {
+	stopCh := make(chan struct{})
```

Comment:
I still don't understand why we need a `stopCh` of our own, wouldn't it work to pass `ctx.Done`?

Reply:
Ah forgot it. Fixed!